src/test/java/com/iexec/common/utils/ContextualLockRunnerTest.java

```diff
@@ -0,0 +1,377 @@
+package com.iexec.common.utils;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.IntStream;
+
+class ContextualLockRunnerTest {
```

Comment:
```suggestion
class ContextualLockRunnerTests {
```
Just for consistency's sake.

Reply:
Sure, thanks!